containers/groovy-examples/docker-compose.yml

```diff
@@ -1,49 +1,35 @@
 version: "3.4"
 
 services:
-  server:
-    image: ghcr.io/deephaven/server:${VERSION:-latest}
-    expose:
-      - '8080'
+  deephaven:
+    image: ghcr.io/deephaven/server:0.16.0-groovy
```

Comment:
I probably owe a "latest" style tag for the jetty images. Ie,

```
ghcr.io/deephaven/server:latest-groovy
ghcr.io/deephaven/server:latest-python
```

Reply:
I assume we want env controls so that we can easily change versions.
```
ghcr.io/deephaven/server:${VERSION:-latest-groovy}
```